src/main/java/org/dataone/hashstore/Client.java

```diff
@@ -0,0 +1,905 @@
+package org.dataone.hashstore;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStreamWriter;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.Statement;
+
+import org.dataone.hashstore.filehashstore.FileHashStoreUtility;
+
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.CommandLineParser;
+import org.apache.commons.cli.DefaultParser;
+import org.apache.commons.cli.HelpFormatter;
+import org.apache.commons.cli.Options;
+import org.apache.commons.cli.ParseException;
+import org.dataone.hashstore.exceptions.HashStoreFactoryException;
+import org.dataone.hashstore.exceptions.PidObjectExistsException;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+
+/**
+ * HashStore's 'Client' class is a command line interface that allows a developer to create a new
+ * HashStore or interact directly with an existing HashStore. See 'README.md' for usage examples.
+ */
+public class Client {
```

Comment:
The name of the class - Client seems too general to me. Also is it really a client? It seems a tool class. May we rename it to something like - CommandLineTool?

Reply:
No problem, it has been renamed.

Reply:
It appears to still be named `Client.java` to me. I agree with @taojing2002 that a more descriptive class name would make sense. Even `CommandLineTool` is too generic. How about `HashStoreClient`?  If we had a shell script to invoke it (or the python version, it would be nice if it were short and mnemonic, like `hashstore`, or `hs` or `hscli`.

Reply:
@mbjones Sorry for the confusion - I hadn't yet pushed my changes, but `HashStoreClient` sounds good to me! I will also create a new issue for hashstore to update the `client.py` module name as well.